ha/setup/ha_setup.py

```diff
@@ -330,6 +339,21 @@ def _get_s3_instance(self, machine_id: str) -> int:
             Log.error(f"Found {s3_instances} which is invalid s3 instance count. Error: {e}")
             raise HaConfigException(f"Found {s3_instances} which is invalid s3 instance count.")
 
+    def _get_mgmt_vip(self, machine_id: str, cluster_id: str) -> dict:
+        mgmt_info = {}
+        try:
+            mgmt_info["mgmt_vip"] = Conf.get(self._index, f"cluster.{cluster_id}.network.management.virtual_host")
+            netmask = Conf.get(self._index, f"server_node.{machine_id}.network.management.netmask")
+            if netmask is None:
+                raise HaConfigException("Detected invalid netmask, It should not be empty.")
+            mgmt_info["mgmt_netmask"] = sum(bin(int(x)).count('1') for x in netmask.split('.'))
+            mgmt_info["mgmt_iface"] = Conf.get(self._index, f"server_node.{machine_id}.network.management.interfaces")[0]
```

Comment:
Do we need to validate the received mgmt_vip and mgmt_iface?

Reply:
Not need, vip resource dose validation
and if invalid, resource creation will fails and config will fail

Reply:
ok